### DISCUSSION TITLE
Add subtotal row at the end of TableRepeater

### DISCUSSION THREAD

**Discussion by Johnrivera7** [awcodes/filament-table-repeater#128] [Q&A]
Hi, @awcodes  thanks for your work on this plugin, I would like to know how I can let a row at the end show me the sums of the rows above, something like totals or subtotals, thank you very much

**Comment by awcodes** [comment]
Well, this is just an abstraction on top of the filament repeater, it's not really a table replacement with the concept of summarization. It's more or less meant for data input vs display. But I would be open to PRs to allow it.
